<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>400</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="menu">
     <widget class="QPushButton" name="newGameButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>New Game</string>
      </property>
     </widget>
     <widget class="QPushButton" name="connectMenuButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>310</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="connectPage">
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="connectToGameButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect!</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="newgame">
     <widget class="QPushButton" name="toGameButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Game!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>220</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>220</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="playersCountLable">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gamePage">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>400</width>
        <height>400</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="scoreLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Score</string>
      </property>
     </widget>
     <widget class="QLabel" name="scoreCountLable">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="gameOverLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>33</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Game Over</string>
      </property>
     </widget>
     <widget class="QLabel" name="firstPlayerNameLable">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="firstPlayerResultsLable">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>160</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="secondPlayerNameLable">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>210</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="secondPlayerResultsLable">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>210</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="thirdPlayerNameLable">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="thirdPlayerResultsLable">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>260</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="foursPlayerNameLable">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>310</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="foursPlayerResultsLable">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>310</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
